src/transformers/models/segformer/configuration_segformer.py

```diff
@@ -81,6 +81,8 @@ class SegformerConfig(PretrainedConfig):
         reshape_last_stage (:obj:`bool`, `optional`, defaults to :obj:`True`):
             Whether to reshape the features of the last stage back to :obj:`(batch_size, num_channels, height, width)`.
             Only required for the semantic segmentation model.
+        semantic_ignore_index (:obj:`int`, `optional`, defaults to 255):
```

Comment:
```suggestion
        loss_ignore_index (:obj:`int`, `optional`, defaults to 255):
```
It's nothing `semantic` that ignores the index, only the loss function.